l10n_fr_fec/wizard/fec.py

```diff
@@ -182,8 +171,7 @@ def generate_fec(self, cr, uid, ids, context=None):
             'type': 'ir.actions.act_window',
             'view_mode': 'form',
             'res_model': self._name,
-            'res_id': ids[0],
+            'res_id': self[0].id,
```

Comment:
Yes, I know. I write it this way because I prefer to make it explicit that I take the first recordet, rather than use the implicit "shortcut".